src/app/content/components/BookBanner.tsx

```diff
@@ -87,27 +108,58 @@ const BookChapter = styled.h1`
 `;
 
 // tslint:disable-next-line:variable-name
-const BarWrapper = styled.div<{theme: Book['theme']}>`
-  position: sticky;
+export const BarWrapper = styled.div<{theme: Book['theme'], up: boolean, variant: 'mini' | undefined}>`
   top: 0;
   padding: 0 ${theme.padding.page.desktop}rem;
   box-shadow: 0 0.2rem 0.2rem 0 rgba(0, 0, 0, 0.1);
   display: flex;
   align-items: center;
-  height: ${bookBannerDesktopHeight}rem;
+  height: ${ifMiniNav(bookBannerDesktopMiniHeight, bookBannerDesktopBigHeight)}rem;
```

Comment:
i had it this way at first, but i was annoyed that the rendered css had all kinds of stuff overriding itself instead of just rendering what it meant

i can put it back this way if you think it is easier to read though